plugins/catalog-react/src/components/UserListPicker/UserListPicker.tsx

```diff
@@ -169,11 +169,16 @@ export const UserListPicker = ({
     ownedFilter.filterEntity(entity),
```

Comment:
Entity ownership filters from query parameters are currently totally clobbered by this component until https://github.com/backstage/backstage/pull/8937 is merged - maybe worth waiting until after that's in to test the changes in this component?

Reply:
Merged now so I’ll resolve this comment!